src/Microsoft.Health.Dicom.Api.UnitTests/Extensions/HttpRequestExtensionsTests.cs

```diff
@@ -67,4 +67,28 @@ public void GivenHttpRequest_WhenGetHostNotFollowingDicomStandard_ThenShouldRetu
         httpRequest.Host.Returns(new HostString(host));
         Assert.Equal(host, httpRequest.GetHost(dicomStandards: false));
     }
+
+    [Fact]
+    public void GivenHttpRequestWithOriginalHeader_WhenGetIsOriginalVersionRequested_ThenShouldReturnExpectedValue()
+    {
+        HttpRequest httpRequest = Substitute.For<HttpRequest>();
+        IHeaderDictionary headers = new HeaderDictionary
+        {
+            { "accept", "application/dicom;msdicom-request-original" }
```

Comment:
interesting that it is part of the accept header media type. Is this custom, should it be a separate custom header?

Reply:
Since users doesnt want to set a value, I included in accept header similar to transfer syntax uid. But thinking again, I could put it in different header property, only thing all the headers follow key value pair. We can pass in true or false.